src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketBuildTrigger.java

```diff
@@ -214,7 +216,16 @@ protected Job asJob() {
         return retrieveScheduleJob(this.job).scheduleBuild2(0,
                 new CauseAction(cause),
                 new ParametersAction(new ArrayList(values.values())),
-                new RevisionParameterAction(cause.getSourceCommitHash()));
+                new RevisionParameterAction(cause.getSourceCommitHash(), getBitbucketRepoUrl(cause.getRepositoryOwner(), cause.getRepositoryName())));
+    }
+
+    private URIish getBitbucketRepoUrl(String repoOwner, String repoName) {
+        try{
+            return new URIish(String.format("git@bitbucket.org:%s/%s.git", repoOwner, repoName));
+        } catch (URISyntaxException e) {
+            logger.log(Level.SEVERE, "Unable to create URIish for bitbucket url, checking out the pull request branch may fail.", e);
+            return null;
```

Comment:
You may want to test returning "" -vs- null. I know this should really really never fail, but, in case it does, wouldn't hurt to verify that a null return won't complicate things down the error path.

Reply:
After looking at this, returning null here will create the RevisionParameterAction similarly to before and subsequent checkouts will fail with the commit not found. If we return a URIish with an empty string the plugin will fail to checkout the branch of the pull request by always ignoring the commit hash. Returning null feels better to me here, let me know which you'd rather have.

Reply:
I'll defer to you for these changes.

I'm going to merge soon, and test with the change(s) required for #180 . Expect a release, if successful, in a week or two.

Reply:
> @drew222 Are you sure this is the correct approach. I just read that we shouldn't be using RPA at all, and, looks like the github plugin uses buildinfo:
> 
> https://github.com/janinko/ghprb/blob/master/src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

Ok - I started trying to do this real quick, and, this plugin is no longer anything like the github one, so I'm not sure the buildinfo will work -- particularly since it is the information from the previous build, which may not exist, and the hash may not be the same. (And you don't seem to be able to specify using buildinfo)